package.json

```diff
@@ -32,5 +32,6 @@
     "commitizen": {
       "path": "./node_modules/cz-conventional-changelog"
     }
-  }
+  },
+  "dependencies": {}
```

Comment:
This will probably disappear after a rebase.

Reply:
Had to manually remove for some reason but I'll push that up in a new commit.